cmd/gpu_plugin/gpu_plugin.go

```diff
@@ -494,6 +510,7 @@ func main() {
 	flag.BoolVar(&opts.resourceManagement, "resource-manager", false, "fractional GPU resource management")
 	flag.IntVar(&opts.sharedDevNum, "shared-dev-num", 1, "number of containers sharing the same GPU device")
 	flag.StringVar(&opts.preferredAllocationPolicy, "allocation-policy", "none", "modes of allocating GPU devices: balanced, packed and none")
+	flag.BoolVar(&opts.exportResourceLabels, "export-resource-labels", true, "export labels from node's GPUs and store them for NFD")
```

Comment:
What would be the benefit for allowing to set the filename? I can only think of duplicate names. That could be circumvented by adding a random postfix to the file.

Reply:
I'm actually leaning towards changing the argument to something like "no-resource-labels" so that default behavior (no parameter) would be to export the resource labels and with the argument it would work as with previous versions.

Reply:
Sounds fine to me.